### ISSUE TITLE
Welcome to contribute!

### ISSUE THREAD

**Issue by daohu527** [ApolloAuto/apollo#15265]
## Contributor
With the release of Apollo 9.0 🚀 , we have taken another small step towards autonomous driving. Of course, we get a lot of questions 🐛 . We hope that **more developers and companies can join in the maintenance of the community**. If you are interested, you can join the [apolloauto-disscuss](https://groups.google.com/g/apolloauto) group. We will regularly discuss and evaluate improvements to Apollo 🍰 .

## Discuss
We have also noticed that online and offline discussions are also very important, so we also plan to organize online technical discussions regularly, as well as offline communication activities. If you are interested or have recommended places, please send an email to daohu527@gmail.com

## Doc
We also need a lot of documentation to introduce and use Apollo, especially some technical articles. If you have good articles, we are more than welcome to learn and promote them. I think this will be of great help to the popularization and development of autonomous driving technology.

**Comment by YuqiHuai** [comment]
@daohu527 If we want to propose a document (technical article), where should we submit it?

**Comment by daohu527** [comment]
> @daohu527 If we want to propose a document (technical article), where should we submit it?

We are considering creating a new documentation repository to meet the requirements. Like other open source projects, I will discuss the contribution details in the near future to let more people participate.

**Comment by maumai-Yubeen** [comment]
@daohu527 
We want to contribute to the Apollo Project as a corporation, just sent you an email regarding the matter!

**Comment by xuming7up** [comment]
We migrate apollo cyberRT on qnx(https://github.com/[xuming7up/cyberQNX](https://github.com/xuming7up/cyberQNX))

Can we create a branch in this repo directly, can contribute the source code?

**Comment by daohu527** [comment]
> We migrate apollo cyberRT on qnx([https://github.com/[xuming7up/cyberQNX](https://github.com/xuming7up/cyberQNX)](https://github.com/%5Bxuming7up/cyberQNX%5D(https://github.com/xuming7up/cyberQNX)))
> 
> Can we create a branch in this repo directly, and contribute the source code?

Yes, you need to specify the environment, chip model, and submit a PR

**Comment by xuming7up** [comment]
> > We migrate apollo cyberRT on qnx([https://github.com/[xuming7up/cyberQNX](https://github.com/xuming7up/cyberQNX)](https://github.com/%5Bxuming7up/cyberQNX%5D(https://github.com/xuming7up/cyberQNX)))
> > Can we create a branch in this repo directly, and contribute the source code?
> 
> @xuming7up Yes, you need to specify the environment, chip model, and submit a PR

@daohu527 
A bit confused with description.
1. We do pull request and merger our code to target branch, and you will help to review.
2. Or submit PR to create new branch for us base on master or other released branch.  Can PR do request new branch?

**Comment by daohu527** [comment]
@xuming7up We can start the discussion here https://github.com/ApolloAuto/apollo/discussions/15563
